web/metarpheus-ts-config.js

```diff
@@ -1,7 +1,16 @@
 const path = require('path');
 const cwd = process.cwd();
 
+const modelPrelude = `// DO NOT EDIT MANUALLY - metarpheus-generated
+import * as t from 'io-ts';
+
+export interface Unit {};
+export const Unit = t.interface({}, 'Unit');
```

Comment:
same here, otherwise this accepts any object

Reply:
same, I hadn't realized it was generated by metarpheus

Reply:
I saw that this is how it is done in Alinity PRO too, is there a reason for that?
https://github.omnilab.our.buildo.io/buildo/alinity-pro/blob/01b8344849910c90aa3977d5e0aafb4cb847bb9c/qia/web/metarpheus-ts-config.js#L54

Reply:
This is actually under my control, it is the config file, but I still would use `interface` to be consistent with the rest of the metarpheus behaviour

Reply:
I think before it was strict by default with `tcomb` but now it's no longer like this with `io-ts`

@giogonzo is this correct?

Reply:
agreed, it'd be better as `strict` in this specific case

Reply:
Fixed!